tests/lacrosse_ltv/LTV-R1/g002_915M_250k.json

```diff
@@ -0,0 +1 @@
+{time : @0.159060s, model : LaCrosse-R1, id : 3670818, seq : 7, flags : 0, rain1 : 170, rain2 : 5226, mic : CRC}
```

Comment:
Make sure the output is valid JSON, this is not valid JSON (e.g. generate with `rtl_433 -F json` with the implemented decoder).